variables/_svgs.pcss

```diff
@@ -149,6 +149,10 @@
 		fill: var(--color-icon-primary);
 		stroke: var(--color-icon-primary);
```

Comment:
let's fix this here, not sure if it's `fill` or `stroke` here but it likely won't be both.